app/views/sessions/new.html.haml

```diff
@@ -0,0 +1,12 @@
+
+%section.sign_in.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = bootstrap_form_for @user,url: sessions_path, html: {class: "form-horizontal"} do |f|
```

Comment:
`, html: {class: "form-horizontal"}` this can be deleted and you can use `bootstrap_form_for` with `layout: :horizontal` - read its docs for more info.